cli/src/live-common-setup-base.js

```diff
@@ -15,6 +15,7 @@ setSupportedCurrencies([
   "litecoin",
   "dash",
   "ethereum_classic",
+  "elrond",
```

Comment:
I don't thinks it is really necesarry to do that, since the currency properties remain the same. We can migrate to testnet/devnet by changing the api endpoint in [env.js](https://github.com/ElrondNetwork/ledger-live-common/blob/elrond-integration/src/env.js).